skimage/measure/_regionprops.py

```diff
@@ -196,9 +196,7 @@ def coords(self):
 
     @only2d
     def eccentricity(self):
-        with warnings.catch_warnings():
```

Comment:
i guess the import warnings statement should be removed too.

Reply:
Good point! done